tools/AksEdgeShell.ps1

```diff
@@ -2,14 +2,22 @@
   AksEdgeShell.ps1
   Validates and loads the config file and imports the bootstrap scripts
 #>
+param(
+    [String] $aideUserConfigfile
+)
+
 #Requires -RunAsAdministrator
-New-Variable -Name gAksEdgeShellVersion -Value "1.0.240904.1500" -Option Constant -ErrorAction SilentlyContinue
+New-Variable -Name gAksEdgeShellVersion -Value "1.0.250313.1500" -Option Constant -ErrorAction SilentlyContinue
 if (! [Environment]::Is64BitProcess) {
     Write-Host "Error: Run this in 64bit Powershell session" -ForegroundColor Red
     exit -1
 }
 
 $aksjson = "$PSScriptRoot\aide-userconfig.json"
+if (-Not [string]::IsNullOrEmpty($aideUserConfigfile)) {

```

Comment:
is this change still required? we can revert back to the original one. Also remove aio-aide-userconfig.json / aio-aksedge-config files?

Reply:
redundant. reverted